### PR TITLE
 Support launch_ros test runner in pytest

### PR DESCRIPTION
**Pull request by hidmic**
This pull request customizes launch tests collection via `pytest` to make use of its own `LaunchTestRunner` whenever a `generate_test_description` is found and it carries a `rostest` marker.

Connected to https://github.com/ros2/launch/pull/312.

### CODE REVIEW

File: launch_testing_ros/test/examples/README.md [hunk 1]
```diff
@@ -1,31 +1,23 @@
 # Examples
 
-## `talker_listener.test.py`
```
Comment (ivanpauno) [inline launch_testing_ros/test/examples/README.md]: Same comment as in https://github.com/ros2/launch/pull/312

Comment (hidmic) [reply launch_testing_ros/test/examples/README.md]: Done in https://github.com/ros2/launch_ros/pull/54/commits/7983a7b238648a181d06128b60406cccd802b125.

**Comment by hidmic** [comment]
All green! Going in.

File: launch_testing_ros/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+### `talker_listener_launch_test.py`
+
+Usage:
+
+```sh
+launch_test test/examples/talker_listener_launch_test.py
+```
+
+This test launches the talker and listener example nodes from demo\_nodes\_py and interacts
+with them via their ROS interfaces.  Remapping rules are used so that one of the tests can sit in
```
Comment (dirk-thomas) [inline launch_testing_ros/README.md]: New sentence should start on new line.

Same below.

Comment (hidmic) [reply launch_testing_ros/README.md]: Oops. It predates the PR but you're right.

**Pull request merged**